pkg/server/config/config.go

```diff
@@ -118,4 +123,17 @@ func modifier(config *CycloneServerConfig) {
 		}
 	}
 
+	if config.SystemNamespace == "" {
+		log.Warningf("SystemNamespace not configured, try to retrieve value from ENV ")
+		config.SystemNamespace = os.Getenv(common.EnvSystemNamespace)
+	}
+}
+
+// validate the cyclone server config
+func validate(config *CycloneServerConfig) error {
+	if config.SystemNamespace == "" {
```

Comment:
Maybe we should fall back to default value `default` and print a warning,  instead of report error.